server/events/policy_check_output_generator.go

```diff
@@ -0,0 +1,125 @@
+package events
+
+import (
+	"fmt"
+
+	"github.com/runatlantis/atlantis/server/events/command"
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/lyft/feature"
+)
+
+type PolicyCheckOutputKey struct {
+	ProjectName string
+	Workspace   string
+}
+
+type PolicyCheckOutputStore struct {
+	Store map[PolicyCheckOutputKey]*models.PolicyCheckSuccess
+}
+
+func NewPolicyCheckOutputStore() *PolicyCheckOutputStore {
+	return &PolicyCheckOutputStore{
+		Store: map[PolicyCheckOutputKey]*models.PolicyCheckSuccess{},
+	}
+}
+
+func (p *PolicyCheckOutputStore) GetOutputFor(projectName string, workspace string) *models.PolicyCheckSuccess {
+	key := PolicyCheckOutputKey{
+		ProjectName: projectName,
+		Workspace:   workspace,
+	}
+
+	if ouptut, ok := p.Store[key]; ok {
+		return ouptut
+	}
+	return nil
+}
+
+func (p *PolicyCheckOutputStore) WriteOutputFor(projectName string, workspace string, policyCheckResult command.ProjectResult) {
```

Comment:
nit: since this is just an internal fxn that updates the store, I would just write this logic in the caller instead of splitting it off to a separate function